changelog.md

```diff
@@ -2,6 +2,7 @@
 
 ## ??? / ???
 
+* Add `import-macros` for more flexible macro module loading (#180)
```

Comment:
Maybe reference #269 here instead/in addition, since it's got a more concise write-up of the intent than the thread in #180?